### DISCUSSION TITLE
Trouble opening new workspaces

### DISCUSSION THREAD

**Discussion by mccabete** [MAAP-Project/Community#794] [Platform] [answered]
When I wen to open a new workspace, I get a `Failed to load the devfile registry URL.`, and get redirected to the "Getting Started Tab" where there are no sample workspaces available. 

<img width="1475" alt="image" src="https://github.com/MAAP-Project/Community/assets/21266199/f38affe8-d3c9-4853-a389-6e54cd81a43c">

<img width="1472" alt="image" src="https://github.com/MAAP-Project/Community/assets/21266199/8c6bed80-3a7c-4ce1-a31a-548604fe7957">

**Comment by sujen1412** [comment, marked as answer]
This might be a transient network issue. 
I was just able to start a workspace. 
Please try again and let us know if this issues still persists.
